README.md

```diff
@@ -119,6 +119,30 @@ apiKey:
   write: WRITE_API_KEY
 ```
 
+### Start a transformation job
+
+`jetfire-cli` also provides a `transform` subcommand, which can start transformation jobs and/or wait for jobs to complete.
```

Comment:
Maybe `run` would have been better but not too important.

Reply:
I realize now that I actually used the word `run` when sketching out the interface. I like that better than `transform` 👍